src/app/modules/admin/components/profile-content/profile-content.component.html

```diff
@@ -0,0 +1,113 @@
+<div class="profile-content">
```

Comment:
i believe we should render template only in case when all needed observables contains data
so, add here smth like ngIf="isReadyToDisplay$ | async" where isReadyToDisplay$ | async is true only if all needed observables contains not null data